presentation/src/main/java/com/anytypeio/anytype/presentation/main/MainViewModel.kt

```diff
@@ -97,6 +98,7 @@ class MainViewModel(
     private suspend fun handleNotification(event: Notification.Event) {
         val payload = event.notification?.payload
         if (payload is NotificationPayload.GalleryImport) {
+            delay(200)
```

Comment:
please export to constant or add comment why we need this delay

Reply:
added a delay, since the notification can come quite quickly, immediately after closing the modal screen with new gallery installation

Reply:
exported